addons/website_slides/models/slide_channel.py

```diff
@@ -53,11 +54,17 @@ def _recompute_completion(self):
             for user in users:
                 users.add_karma(partner_karma[user.partner_id.id])
 
-    def unlink(self):
-        """
-        Override unlink method :
-        Remove attendee from a channel, then also remove slide.slide.partner related to.
-        """
+    @api.model
+    def create(self, values):
+        res = super(ChannelUsersRelation, self).create(values)
+        slide_partners = self.env['slide.slide.partner'].sudo().search([
+            ('channel_id', '=', res.channel_id.id),
+            ('partner_id', '=', res.partner_id.id)
+        ])
+        res.filtered(lambda cp: cp.partner_id in slide_partners.partner_id)._recompute_completion(False)
```

Comment:
small comment on False ? (just one line to tell : we are doing "that")
Also, name the parameter (as it's a named one)